src/containers/Bookings.js

```diff
@@ -1,19 +1,37 @@
-import React, { Component } from 'react';
-import Search from '../components/Search.js';
+import React, { Component } from "react";
+import Search from "../components/Search.js";
 // import Results from '../components/Results.js';
-// import FakeBookings from '../data/fakeBookings.json';
+import Results from "../components/Results.js";
 
+// import FakeBookings from '../data/fakeBookings.json';
+import allBookings from "../data/fakeBookings.json";
 export default class Bookings extends Component {
-  search = () => {
-    console.info('TO DO!');
+  updateBookingsToDisplay = (input) => {
+    this.setState({ bookingsToDisplay: allBookings.filter(x => x.id === parseInt(input, 10))})
   };
-  
+  updateBookingsToDisplay2 =(input) =>{
```

Comment:
To make it easier for people to read and understand what this function is doing, I would rename it to something like `updateBookingsToDisplayByName`, and the function above to something like `updateBookingsToDisplayById`.

Reply:
thanks a lot @40thieves for reviewing my PR. Really we can not thank you enough for your efforts.